src/modules/Modules.cpp

```diff
@@ -61,7 +62,7 @@ void setupModules()
         traceRouteModule = new TraceRouteModule();
         neighborInfoModule = new NeighborInfoModule();
         detectionSensorModule = new DetectionSensorModule();
-
+        atakPluginModule = new AtakPluginModule();
```

Comment:
Maybe we can only enable it for the `TAK` role to enforce people using that for even more optimized settings?

Reply:
I had wondered if we might do that as well. I'm somewhat on the because I'm not sure what value it holds to gate it off with the role, since the module should ignore traffic outside of its portnum otherwise. anyway

Reply:
I see, that's true.